llvm/test/Transforms/LoopVectorize/select-fmax-last-index.ll

```diff
@@ -0,0 +1,132 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --version 5
+; RUN: opt -passes=loop-vectorize -force-vector-width=4 -force-vector-interleave=1 -S %s | FileCheck %s
+
+define i64 @test_vectorize_select_fmaxnum_idx(ptr %src, i64 %n) {
```

Comment:
May be good to continue and indicate if the desired `idx` is first or last, here and below.

Reply:
Done, thanks